ocaml/parsing/jane_syntax_parsing.mli

```diff
@@ -257,6 +257,12 @@ module Constructor_argument :
        and type ast_desc = Parsetree.core_type_desc With_attributes.t
        and type ast_info = Parsetree.attributes
 
+module Extension_constructor :
+  AST with type ast = Parsetree.extension_constructor
+       and type ast_desc =
+             Parsetree.extension_constructor_kind With_attributes.t
+       and type ast_info = Parsetree.attributes * string Location.loc
```

Comment:
Is it worth commenting what the `string Location.loc` is? I think so: it's different than the other `ast_info` types in this file.